esrally/rally.py

```diff
@@ -651,6 +651,7 @@ def add_track_source(subparser):
     )
     race_parser.add_argument(
         "--user-tag",
+        "--user-tags",
```

Comment:
didn't change the order to keep backward compatibility, in order to be able to set tag by env var `USER_TAG`.